apps/src/music/views/PatternAiPanel.tsx

```diff
@@ -0,0 +1,380 @@
+import classNames from 'classnames';
+import React, {
+  ChangeEvent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+const aiBotImages = [
+  require(`@cdo/static/music/ai/ai-bot-0.png`),
+  require(`@cdo/static/music/ai/ai-bot-1.png`),
+  require(`@cdo/static/music/ai/ai-bot-2.png`),
+  require(`@cdo/static/music/ai/ai-bot-3.png`),
```

Comment:
I think you can `import` these (and construct the array from the imported modules)